src/libostree/ostree-kernel-args.c

```diff
@@ -21,6 +21,7 @@
 
 #include "ostree-kernel-args.h"
 #include "libglnx.h"
+#include "../libotutil/ot-tool-util.h"
```

Comment:
I actually ran into the problem including `ot-tool-util.h` once before, but ended up not needing to use it.
We could do this: https://github.com/rfairley/ostree/commit/b75946dc526306f48a3630e68e787ba09f80ecc4 and then `#include "otutil.h"` here (other libostree code also includes `otutil.h`).